src/test/java/configuration/CognitoSettingsTest.java

```diff
@@ -0,0 +1,35 @@
+package configuration;
+
+import io.micronaut.test.annotation.MicronautTest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import javax.inject.Inject;
+
+
+@MicronautTest(propertySources = {"classpath:application-configuration.yaml"})
+class CognitoSettingsTest {
+
+	@Inject
+	CognitoSettings cognitoSettings;
+
+	@Test
+	void getRegion() {
+		Assertions.assertEquals("eu-west-2", cognitoSettings.getRegion());
```

Comment:
Why do these tests? (Testing configuration against hard-coded values)

Reply:
I suppose to check in the test case is whether we will able to connect Cognito or not.

As of now, I am just validating that are we getting values which are configured in the application-test.yaml. sooner later the above test will be replaced with testing AWS Cognito